tests/improg_prog_confirm.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# add 2019-04-04 by Philippe Duveau, released under ASL 2.0
+. ${srcdir:=.}/diag.sh init
+generate_conf
+add_conf '
+module(load="../contrib/improg/.libs/improg")
+input(type="improg" tag="tag" ruleset="ruleset"
+	  binary="./improg-simul.sh -e '$RSYSLOG_DYNNAME'.stderr -c -n 1"
+	  confirmmessages="on" signalonclose="off" killunresponsive="on"
+	 )
+ruleset(name="ruleset") {
+	action(type="omfile" file="'$RSYSLOG_OUT_LOG'")
+}
+'
+startup
+shutdown_when_empty
+wait_shutdown
+content_check "program data"
+if [ ! -e $RSYSLOG_DYNNAME.stderr ]; then
+	echo $RSYSLOG_DYNNAME'.stderr missing'
+	error_exit 1
+fi
+export EXPECTED='START Received
+ACK Received
+STOP Received'
+echo "$EXPECTED" | cmp - $RSYSLOG_DYNNAME.stderr
```

Comment:
you should replace this and the if by `cmp_exact` - I can also change it as part of the merge.

Reply:
Ok, I did not have a look to the function. I was convincted that cmp_exact was only for the standard out log file. I change that now. Thx

Reply:
Done in last PR